src/main/java/signup_and_login/UserModel.java

```diff
@@ -0,0 +1,21 @@
+package signup_and_login;
+
+public class UserModel {
```

Comment:
I think that UserModel (and User entity) should be updated to include a userID. This is then used to denote room hosts and compare users. This uuid would be randomly generated when a new user is created, and then the user updated with a setter.

Reply:
see commit 447b882605d4d15fc8cb8bd701de977f6cbfa13e